imports.md

```diff
@@ -1190,6 +1190,10 @@ implicitly bind the socket.</p>
 <li><code>not-in-progress</code>:           A <code>bind</code> operation is not in progress.</li>
 <li><code>would-block</code>:               Can't finish the operation, it is still in progress. (EWOULDBLOCK, EAGAIN)</li>
 </ul>
+<h1>Implementors note</h1>
+<p>When binding to a non-zero port, this bind operation shouldn't be affected by the TIME_WAIT
+state of a recently closed socket on the same local address (i.e. the SO_REUSEADDR socket
+option should be set implicitly on platforms that require it).</p>
```

Comment:
It looks like Windows' `SO_REUSEADDR` has [different and undesirable behavior](https://docs.microsoft.com/en-us/windows/win32/winsock/using-so-reuseaddr-and-so-exclusiveaddruse) so it'd be good to call out that this advice doesn't apply on Windows.

Reply:
Done!